src/components/Map/MapboxWrapper.js

```diff
@@ -122,29 +139,25 @@ class MapboxWrapper {
     let v = [];
     if (this.vehicles) {
       const oldPositions = _.keyBy(this.vehicles, 'id');
-      v = vehicles.map((vehicle) => {
-        return {
-          id: vehicle.id,
-          route: vehicle.route.replace('1_', ''),
-          lastPosition: oldPositions[vehicle.id] ? oldPositions[vehicle.id].currentPosition : vehicle.coords,
-          currentPosition: oldPositions[vehicle.id] ? oldPositions[vehicle.id].currentPosition : vehicle.coords,
-          nextPosition: vehicle.coords,
-        };
-      });
+      v = vehicles.map((vehicle) => ({
+        id: vehicle.id,
+        route: vehicle.route.replace('1_', ''),
```

Comment:
This isn't gonna work for other cities. Especially cities with multiple agencies. 1_ happens to be the agency is for Austin. Take a look at the Tampa or Atlanta OBA servers.